test/e2e/specs/editor/various/style-variation.spec.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'adding blocks', () => {
+	test( 'Should switch to the plain style of the quote block', async ( {
+		page,
+		pageUtils,
+	} ) => {
+		await pageUtils.createNewPost();
+
+		// Inserting a quote block
+		await pageUtils.insertBlock( {
```

Comment:
Hey, as far as I understand, this PR changed the original test behaviour. Previously, we:

1. added a block
2. typed some content

These steps covered the case for which we wanted to make sure that users could type right after the block was inserted.

The playwright test does this instead:

1. adds a block with the contents we want

It doesn't look like we're testing for typing. Sorry, I'm not very familiar with playwright, but shouldn't we respect the existing steps when migrating tests? @JustinyAhin @kevin940726